libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -86,6 +88,7 @@
 #define FLOW_M_NSE_DEFAULT      0.25f
 #define FLOW_I_GATE_DEFAULT     300
 #define CHECK_SCALER_DEFAULT    150
+#define FLOW_USE_MASK           2
```

Comment:
name the define FLOW_USE_MASK_DEFAULT?

Reply:
Also potted that the unknown vehicle type is defaulted to 3. It should be the same as copter. Will fix that also.